azure/iac/networkwatchersflowlogs.rego

```diff
@@ -28,25 +34,27 @@ azure_issue["netwatch_logs"] {
 
 netwatch_logs {
     lower(input.resources[_].type) == "microsoft.network/networkwatchers/flowlogs"
-    not azure_issue["netwatch_logs"]
     not azure_attribute_absence["netwatch_logs"]
+    not azure_issue["netwatch_logs"]
 }
 
 netwatch_logs = false {
-    azure_issue["netwatch_logs"]
+    azure_attribute_absence["netwatch_logs"]
 }
 
 netwatch_logs = false {
+    azure_issue["netwatch_logs"]
+}
+
+netwatch_logs_miss_err = "NetWatcher FlowLog extension attribute retentionPolicy missing in the resource" {
```

Comment:
Network watchers flowlog attribute 'enabled' is missing from the resource.

Reply:
fixed